web/src/tools/testing/bulk_rendering/build.sh

```diff
@@ -17,10 +16,9 @@ cd "$THIS_SCRIPT_PATH"
 
 ################################ Main script ################################
 
-builder_describe \
+builder_describe "Builds a 'bulk renderer' that loads all the cloud keyboards from api.keyman.com and renders each of them to a document." \
   "@/web/src/app/browser build" \
   "@/web/src/app/ui build" \
-  "Build the bulk renderer project. The bulk renderer loads all the cloud keyboards from api.keyman.com and renders each of them to a document." \
   "clean" \
```

Comment:
Huh, how did this slip through before? How did it even work properly?!

Reply:
Seems like builder just silently ignored it?  But yeah, I thought that odd.

Reply:
It ended up being a possible action I guess, called 'Build'